app/services/analytics_events.rb

```diff
@@ -248,4 +248,29 @@ def proofing_address_result_missing
   def proofing_document_result_missing
     track_event('Proofing Document Result Missing')
   end
+
+  # @identity.idp.event_name SAML Auth Request
+  # @param [Boolean] idv indicating whether identity verification is needed
+  # @param [Boolean] finish_profile indicating if proofing is needed for a pending/reset profile
```

Comment:
can we rename this to `profile_needs_verification`? or `proofing_is_needed`? The name doesn't make sense to me without the comment

Reply:
This was copied from the existing SAML event's attributes, but I do agree and will change it for this one at least

Reply:
Fixed in [8914669](https://github.com/18F/identity-idp/pull/6155/commits/89146691bdf2598240a2e930ab2d903420b87ca4)